CHANGELOG.md

```diff
@@ -4,12 +4,15 @@
 
 ### Added
 
+- kedro-mlflow hooks are now auto-registred (#29)
```

Comment:
We should not do this in this PR, since it breaks kedro viz (see https://github.com/quantumblacklabs/kedro-viz/issues/260). Can you remove it?

Reply:
kedro-viz just work fine with kedro 0.16.5 and auto registred hooks. I'll test it with previous kedro versions